py/kubeflow/tf_operator/tf_job_client.py

```diff
@@ -418,6 +419,61 @@ def get_start_time_by_index(api_client, namespace, name, replica_type,
                                            replica_index, phase)
 
 
+def get_streaming_log(api_client, namespace, pod_name, container_name,
+    wait_time):
+  """Stream log from container.
+
+  Args:
+    api_client: The K8s API client.
+    namespace: The K8s namespace.
+    pod_name: Pod name to stream log.
+    container_name: Container name to stream log.
+    wait_time: Time between adjacent read_namespaced_pod_log operations.
+  """
+  core_api = k8s_client.CoreV1Api(api_client)
+  prev_time = time.time()
+  while True:
```

Comment:
When will the loop end?

Reply:
The loop will end with the Python program, otherwise it will keep fetching the logs.

Reply:
Does it meet the demand of kubeflow pipelines? if it does, then LGTM

Reply:
I think so. The Kubeflow pipeline will launch tfjob inside a container (please see [here](https://github.com/kubeflow/pipelines/blob/848d4fb99c9dc3e37012705a3468b27fe66193e1/components/kubeflow/launcher/src/launch_tf_job.py)). After the container exits, the loop will end accordingly. 

I haven't seen the need to expose additional control to stop the logging thread earlier. If needed, we can add a threading.Event to stop loop, and give this control to users.

Reply:
SGTM